### DISCUSSION TITLE
Migrating user data from rooted device A to non-rooted device B

### DISCUSSION THREAD

**Discussion by cmonty14** [bromite/bromite#2506] [Q&A]
Hi,
I have found [this](https://github.com/bromite/bromite/wiki/Migrating-user-data) wiki article for migrating user data.

The procedure for fetching data from device A is clear; this is simple if device (OnePlus 5T) is rooted.

However, I don't understand how to store user data in device B that is not rooted (Google Pixel 6a).

Can you please advise how to complete the migration task?

THX

**Comment by jan449** [comment]
I don't know it is possible to bringing datas from the root  file system to an non rooted devicement! I mean not
